internal/model/user.go

```diff
@@ -41,6 +46,7 @@ type User struct {
 	Permission int32  `json:"permission"`
 	OtpSecret  string `json:"-"`
 	SsoID      string `json:"sso_id"` // unique by sso platform
+	Authn      string `gorm:"size:4294967295" json:"-"`
```

Comment:
Why does it need to be so long?

Reply:
> Why does it need to be so long?

even one credential takes around 1000 characters, to be honest, now I also think it's a bit too long, how about let's shorten it to 65535(MySQL Largest)?

Reply:
I think you can use `text` type directly like https://github.com/alist-org/alist/blob/0b675d6c024cbaec31807113b5a65f551a289d80/internal/model/storage.go#L12

Reply:
> gorm:"type:text"

got it, i tested it and type text works fine